EIPS/eip-2027.md

```diff
@@ -0,0 +1,74 @@
+---
+eip: 2027
+title: State Rent C - Net contract size accounting
+author: Alexey Akhunov (@AlexeyAkhunov)
+discussions-to: https://ethereum-magicians.org/t/eip-2027-net-contract-size-accounting-change-c-from-state-rent-v3-proposal/3275
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-05-14
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+Ethereum starts counting the number of storage slots filled and emptied in the contracts. Since the number of pre-existing slots is not currently accounted
+in the state, effectively, only net change in the number of slots is tracked. In the subsequent change, called *Gross contract size accounting*, the total
+number of storage slots starts being tracked.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+This is part of the State Rent roadmap. This particular change introduced initial, net accounting of the number of the contract storage slots. Though not very
```

Comment:
I think this should be `This particular change introduces`

Reply:
Done, thank you!